controls/radwindow/how-to/hide-minimize-maximize-buttons.md

```diff
@@ -35,7 +35,33 @@ The easiest way to hide the minimize or maximize buttons is to set the __HideMin
 
 ## Use the ResizeMode property
 
-By setting the __ResizeMode__ property to __CanMinimize__ only the __Minimize__ button will be visible. The resize button will get hidden and the resizing will get disabled. By setting the __ResizeMode__ property to __NoResize__ both the Minimize and the Maximize buttons will get hidden, the resizing will also get disabled. 
+By setting the __ResizeMode__ property to __CanMinimize__ only the __Minimize__ button will be visible. The resize button will get hidden and the resizing will get disabled. This is demonstrated in __Example 2__.
+
+#### __[XAML] Example 2: Showing only the Minimize Button__
+
+{{region xaml-radwindow-how-to-hide-minimize-maximize-buttons-0}}

```

Comment:
We should update the code in the 3 examples to match result in the pictures (by including the TextBlock content of the RadWindow).